src/app/assessment/assessment.component.html

```diff
@@ -53,8 +55,11 @@ <h3>{{ group.name }}</h3>
                 [doReview]="doReview"
                 [submission]="(submission.answers && submission.answers[question.id]) ? submission.answers[question.id] : {}"
                 [review]="(review.answers && review.answers[question.id]) ? review.answers[question.id] : {}"
+                [reviewStatus]="review.status"
+                [submissionStatus]= "submission.status"
                 [formControlName]="'q-' + question.id"
                 [control]="questionsForm.controls['q-' + question.id]"
+                (saveProgress)="submit($event)"
```

Comment:
Since `$event` will always return `true`, we can just put `true` here instead of `$event`. Same as below